cmd/tendermint/commands/run_node.go

```diff
@@ -89,6 +90,23 @@ func AddNodeFlags(cmd *cobra.Command) {
 		"db-dir",
 		config.DBPath,
 		"database directory")
+
+	cmd.Flags().String(
+		"ipfs.repo-path",
+		config.IPFS.RepoPath,
+		"custom IPFS repository path. Defaults to `.{RootDir}/ipfs`",
+	)
+	cmd.Flags().Bool(
+		"ipfs.serve-api",
+		config.IPFS.ServeAPI,
+		"set this to expose IPFS API(useful for debugging)",
+	)
+	cmd.Flags().BoolVar(
+		&ipfs.EmbeddedInit,
```

Comment:
Any reason this is a global var?

Reply:
Yeah, because it is used between packages.